2996_reflection/reflection.md

```diff
@@ -5734,6 +5736,38 @@ int a = fn<^^NS>();
 :::
 :::
 
+### 13.8.3.7 [temp.dep.concept] Dependent concepts {-}
+
+Add a new section to cover dependent concepts.
+
+::: std
+:::addu
+**Dependent concepts   [temp.dep.concept]**
+
+[1]{.pnum} A concept is dependent if it is designated by a dependent `$splice-specifier$`. A `$concept-name$` is dependent if its lookup context is dependent.
+
+[2]{.pnum} The program is ill-formed if any construct names or designates a dependent concept.
+
+::: example
+```cpp
+template <std::meta::info C, std::meta::info N>
+struct S {
+  void f() requires template [:C:]<S> {}       // error: concept designated by dependent splice
+  void g() requires [:N:]::template C<S> {}    // error: dependent concept-name
```

Comment:
Do you think it's worth adding an example like...
```cpp
bool h() {
    return extract<bool>(substitute(C, {^^int})); // ok, returns true
}
```
Help illustrate that the issue is dependent splicing, not literally any dependent use at all.